core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -156,6 +158,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return getResult.toXContent(builder, params);
```

Comment:
also MultiGetResponse was using this, but I just removed the startObject and endObject from it.